projects/ngx-braintree/src/lib/ngx-braintree.component.ts

```diff
@@ -245,42 +247,59 @@ export class NgxBraintreeComponent implements OnInit, OnDestroy, OnChanges {
 
   pay(): void {
     if (this.instance) {
+      this.processingStatus.next(true);
+
       this.instance.requestPaymentMethod({
         threeDSecure: this.dropinConfig.threeDSecure,
-      }, (err, payload) => {
+      }, async (err, payload) => {
         if (err) {
           console.error(err);
           this.errorMessage = err;
+          this.processingStatus.next(false);
```

Comment:
@VeselyT lets use `.emit()` instead, as its preffered way in case of `EventEmitter`.